### ISSUE TITLE
Can't parse macCatalyst API availability annotation

### ISSUE THREAD

**Issue by nornagon** [ziglang/zig#9978]
### Zig Version

0.9.0-dev.1414+cde3dd365

### Steps to Reproduce

1. Have the MacOSX 10.15 SDK
2. Pull these files: https://gist.github.com/nornagon/02fb285f4a0038c407b3a6531cd79321
3. `zig build`

### Expected Behavior

No errors when including CFURL.h

### Actual Behavior

```
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CFURL.h:1159:83: error: expected ','
```

That line is:

```
    kCFURLBookmarkCreationWithSecurityScope API_AVAILABLE(macos(10.7), macCatalyst(13.0))  API_UNAVAILABLE(ios, watchos, tvos) = ( 1UL << 11 ), // Mac OS X 10.7.3 and later, include information in the bookmark data which allows the same sandboxed process to access the resource after being relaunched
```

I think it's the `macCatalyst` token which it's not able to parse?

**Comment by andrewrk** [comment]
Looks like it might be fixed by 2006add8496c47804ee3b6c562f420871cb4ea0a.

**Comment by nornagon** [comment]
Looks like this fixed the issue! Thanks @andrewrk, I can now use zig to compile glfw on macOS :)
